### PR TITLE
Adds some Skrell weapons/armour/uniforms for the Skrell team.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Title. Includes a rig, uniforms, a telebaton, some guns.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

**Review by Alberyk — APPROVED** [review]

File: code/game/objects/items/weapons/swords_axes_etc.dm [hunk 1]
```diff
@@ -100,4 +105,16 @@
 				var/mob/living/carbon/human/T = target
 				T.apply_damage(40, PAIN, target_zone)
 		return
-	return ..()
\ No newline at end of file
+	return ..()
+
+/obj/item/melee/telebaton/nlom
+	name = "nlomkala baton"
```
Comment (listerla) [inline code/game/objects/items/weapons/swords_axes_etc.dm:111]:
```suggestion
	name = "\improper Nlomkala baton"
```

File: code/modules/clothing/spacesuits/rig/suits/skrell.dm [hunk 2]
```diff
@@ -47,16 +47,19 @@
 /obj/item/clothing/head/helmet/space/rig/skrell/tup
 	light_overlay = "helmet_light_tadpole"
 
+/obj/item/clothing/head/helmet/space/rig/skrell/tup/modern
+	light_overlay = "tup2_rig_helmetlight"
+
 /obj/item/rig/skrell/tup
-	name = "tup suit control module"
-	desc = "A suit control module designed by the Jargon Federation for Tup operations. The best of the best."
+	name = "outdated tupkala suit control module"
```
Comment (listerla) [inline code/modules/clothing/spacesuits/rig/suits/skrell.dm:54]:
```suggestion
	name = "outdated Tupkala suit control module"
```

**Comment by listerla** [comment]
Actually, I think the gist is across so suggestions will stop as to not spam. Point is all the Kala branches are proper nouns [according to the wiki.](https://wiki.aurorastation.org/index.php?title=Skrell_Politics#Federation_Enforcement)

**Comment by NonQueueingMatt** [comment]
The nouns in the name fields are intentionally lowercased.

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**